common/geometry/opencv_viz.hh

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <string>
+#include <vector>
+
+#include "Eigen/Core"
+#include "Eigen/Geometry"
+#include "opencv2/viz.hpp"
```

Comment:
I think this can move into `opencv_viz.cc` since it's not required for the interface.